docs/src/ApiClient.js

```diff
@@ -0,0 +1,690 @@
+/**
+ * Ambianic Serverless functions API Definition
+ * An API Definition for Ambianic Serverless functions created using Swagger
+ *
+ * The version of the OpenAPI document: 0.1
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ *
+ */
+
+
+import superagent from "superagent";
+import querystring from "querystring";
+
+/**
+* @module ApiClient
+* @version 0.1
+*/
+
+/**
+* Manages low level client-server communications, parameter marshalling, etc. There should not be any need for an
+* application to use this class directly - the *Api and model classes provide the public API for the service. The
+* contents of this file should be regarded as internal but are documented for completeness.
+* @alias module:ApiClient
+* @class
+*/
+class ApiClient {
+    constructor() {
+        /**
+         * The base URL against which to resolve every API call's (relative) path.
+         * @type {String}
+         * @default https://ambianic-functions.netlify.app
+         */
+        this.basePath = 'https://ambianic-functions.netlify.app'.replace(/\/+$/, '');
```

Comment:
@vickywane I notice here that the client SDK js uses a hardcoded base URL for the API. 
`https://ambianic-functions.netlify.app`

How can we run tests against a local CI version of the API and a test run against a netlify preview deployment *before* we push a release.

It does not make sense to run the CI tests against an already deployed API version. That's backwards.

Reply:
We probably need to update this URL dynamically with:

* localhost:8080/ when running tests against a local netlify dev node instance to test functions
* ambianic-functions-preview.netlify.app/ (actual URL value provided by the netlify preview deploy github action) when running tests against a preview deployment

Reply:
@ivelin This has been implemented now. 

However, in two out of three of the postman tests, we are using a mock server that returns exactly the same response object Stripe returns. In those cases, the mock server URL is still in use.

Reply:
@vickywane do we need both mocha and postman?

Reply:
We are dropping auto-generated mocha tests in favour of Postman.